tests/data/packages/README.txt

```diff
@@ -104,3 +104,23 @@ require_simple-1.0.tar.gz
 ------------------------
 contains "require_simple" package which requires simple>=2.0 - used for testing
 if dependencies are handled correctly.
+
+invalid_version-2010i-py3-none-any.whl
+--------------------------------------
+The invalid-version package with a legacy version.
+
+invalid_version-1.0-py3-none-any.whl
+------------------------------------
+A valid variant of the invalid-version package.
```

Comment:
Probably call it `valid_version` then?

Reply:
It's the same package name as `invalid-version`, but with valid metadata.

Reply:
Let’s add a sentence to explain why this file is called this.

Reply:
I added a sentence giving an example of test using this package.